thoth/metrics_exporter/jobs.py

```diff
@@ -82,16 +82,55 @@ def countGraphSyncJobStatus(JobListItems: dict) -> (int, int, int):
 
 def get_thoth_graph_sync_jobs():
     """Get the total number of graph-sync Jobs per category with corresponding status."""
-    namespace = os.getenv("MY_NAMESPACE")
+    namespaces = []
+    namespaces.append(os.getenv("MY_NAMESPACE"))
+
+    if os.getenv("THOTH_MIDDLETIER_NAMESPACE"):
+        namespaces.append(os.getenv("THOTH_MIDDLETIER_NAMESPACE"))
```

Comment:
It might be a good idea to log a warning (using logging) if namespaces are missing and info if the namespace will be monitored.